packages/augur-ui/src/modules/sitemap/react-sitemap.js

```diff
@@ -0,0 +1,30 @@
+require('@babel/register');
+const fs = require('fs');
+
+const router = require('./routes').default;
+const Sitemap = require('react-router-sitemap').default;
+
+const address =
+  process.env.NODE_ENV === 'development'
+    ? 'https://v2.augur.net/#!'
+    : `https://cloudflare-eth.com/${process.env.IPFS_STABLE_LOADER_HASH ||
+        'QmVfCcpugkz2WRCWJ75jnDSKAGXAQEgV8XEd839An6aXLE'}/#!`;
+
+console.log(`Using the following base address: "${address}".`);
```

Comment:
is this needed ?

Reply:
Not needed but might help to quickly identify when something is wrong with the sitemap or the stable loader hash